socialhome/users/serializers.py

```diff
@@ -63,9 +76,20 @@ class Meta:
             "image_url_medium",
             "image_url_small",
             "is_local",
+            "pinned_content_exists",
+            "safer_image_url_large",
             "url",
         )
 
+    def get_following_count(self, obj):
+        return str(obj.following.count())
+
+    def get_followers_count(self, obj):
+        return str(Profile.objects.followers(obj).count())
+
+    def get_pinned_content_exists(self, obj):
+        return Content.objects.profile_pinned(obj, obj.user).filter(pinned=True).exists()
```

Comment:
`obj.user` should be the user *viewing the page*, not the user linked to profile. The `Content` queryset uses this to check what content is visible. For example, the profile might have pinned content which is not visible to the viewing user.

See [this serializer method in Content serializer](https://github.com/jaywink/socialhome/blob/bf8dc7af5a4cfe4f4d8e73aa6aa2464446818961/socialhome/content/serializers.py#L83) on how to get the user from the request object.

Reply:
No need for `pinned=True`, the `profile_pinned()` queryset method already does that.